packages/server/tests/acceptance/htsPrecompile/tokenCreate.spec.ts

```diff
@@ -638,45 +637,32 @@ describe('@tokencreate HTS Precompile Token Create Acceptance Tests', async func
         nftTransfers: [],
       }];
 
-      try {
-        const txXfer = await mainContract.cryptoTransferPublic(tokenTransferList);
-        expect((await txXfer.wait()).events.filter(e => e.event === Constants.HTS_CONTRACT_EVENTS.ResponseCode)[0].args.responseCode).to.equal(TX_SUCCESS_CODE);
-      } catch (error: any) {
-        expect(error.code).to.equal(Constants.CALL_EXCEPTION);
-        expect(error.reason).to.equal("transaction failed");
-      }
+      await Assertions.expectRevert(mainContract.cryptoTransferPublic(tokenTransferList), Constants.CALL_EXCEPTION);
```

Comment:
Q: is the reason no longer important to check or is it always the same?

Reply:
I tried to remove all of the copy-paste hardcoded checks and to add centralized asserts (some kind of mini refactoring).